docs/resources/as_group.md

```diff
@@ -145,18 +123,19 @@ The following arguments are supported:
 * `desire_instance_number` - (Optional, Int) Specifies the expected number of instances. The default value is the
   minimum number of instances. The value ranges from the minimum number of instances to the maximum number of instances.
 
-* `min_instance_number` - (Optional, Int) Specifies the minimum number of instances. The default value is 0.
+* `min_instance_number` - (Optional, Int) Specifies the minimum number of instances. Defaults to **0**.
 
-* `max_instance_number` - (Optional, Int) Specifies the maximum number of instances. The default value is 0.
+* `max_instance_number` - (Optional, Int) Specifies the maximum number of instances. The value ranges from **0** to **300**.
+  Defaults to **0**.
 
-* `cool_down_time` - (Optional, Int) Specifies the cooling duration (in seconds). The value ranges from 0 to 86400,
-  and is 300 by default.
+* `cool_down_time` - (Optional, Int) Specifies the cooling duration (in seconds). The value ranges from **0** to **86400**.
```

Comment:
The value ranges from `0` to `86,400`.

Reply:
Please also check the relevant formats in other places

Reply:
OK